tools/topology/topology2/cavs/platform/intel/dmic-generic-add-dmic1.conf

```diff
@@ -0,0 +1,237 @@
+Object.Dai {
```

Comment:
Not following the dmic1 in the file name ?

Reply:
Yes, not too descriptive, would dmic-generic-add-dmic16k.conf be better?

Reply:
yes